features/step_definitions/ocm.rb

```diff
@@ -1,3 +1,4 @@
+require 'webauto/webconsole_executor'
```

Comment:
Is this what we indeed need to modify?

Reply:
This is added in yuwan's PR in https://github.com/xueli181114/verification-tests/pull/24/files.
Without this line, our code cannot run in Jenkins.